packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.service.js

```diff
@@ -0,0 +1,28 @@
+import { PRODUCT_TYPES } from './ipv6.constant';
+
+export default class IpAgoraV6Order {
+  /* @ngInject */
+  constructor($q, $http, OvhHttp, IpAgoraOrder) {
+    this.$q = $q;
+    this.$http = $http;
+    this.OvhHttp = OvhHttp;
+    this.IpAgoraOrder = IpAgoraOrder;
+  }
+
+  getVrackService() {
+    return this.OvhHttp.get('/products', {
```

Comment:
use `$http` instead of `OvhHttp`. Pay attention, the parameters will change a bit